lesson2/lesson2_def.py

```diff
@@ -1,62 +1,61 @@
 import random
 
 
-if __name__ == "__main__":
-# 1
-    def get_avarage(list):
-        list_digit = list.split(",")
-        su = 0
-        for dig in list_digit:
-            print(dig)
-            su=su+int(dig)
-        return su/len(list_digit)
+def get_avarage(list):
+    list_digit = list.split(",")
+    su = 0
+    for dig in list_digit:
+        print(dig)
+        su = su + int(dig)
+    return su / len(list_digit)
 
 
-    print(get_avarage(input('Input string of digit separatet by ,')))
-# 2
-    array_of_input = [12, 601, 43, 67, 101, 34]
+def get_summ_digit(digit):
+    st = str(digit)
+    sum_d = 0
+    for i in st:
+        sum_d += int(i)
+    return sum_d
+
 
+def sort(array):
+    fl = False
+    sort_sum = []
```

Comment:
Variable should start with noun, not with verb.
```suggestion
    sorted_sum = []
```